Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Constants.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Contoso.Forms.Demo
+{
+    [Android.Runtime.Preserve(AllMembers = true)]
```

Comment:
Was the symbol removed in release?, it's suprising as there is no reflection here so should not be necessary.

Did you actually test in release and had a crash when this attribute was missing on this constant?

Reply:
@guperrot I guess it's just to make the things the same as here
https://github.com/microsoft/appcenter-sdk-dotnet/pull/1027/files#diff-3be604fc8d5e709029be52313f8b20b3R6

Reply:
Let's not introduce the annotation if not necessary, please run locally in release and verify.